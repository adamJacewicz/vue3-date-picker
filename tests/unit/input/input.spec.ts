import { shallowMount } from '@vue/test-utils';
import CInput           from '@/components/atoms/input.vue';

describe('Input', () => {
  const input = shallowMount(CInput, {
    props: {
      value: 'Mock!',
    },
  });

  it('render component with correct class', () => {
    expect(input.classes()).toContain('c-input__label');
  });

  it('store correct value', () => {
    expect(input.props('value')).toEqual('Mock!');
  });

  it('should match snapshot', () => {
    expect(input.html()).toMatchSnapshot();
  });
});
