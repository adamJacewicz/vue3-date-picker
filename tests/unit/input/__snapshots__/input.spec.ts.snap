// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input should match snapshot 1`] = `<label class="c-input__label"> <input class="c-input"></label>`;
